common/src/main/java/com/redhat/cloud/notifications/openbridge/BridgeHelper.java

```diff
@@ -0,0 +1,107 @@
+package com.redhat.cloud.notifications.openbridge;
+
+import io.quarkus.cache.CacheResult;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+
+import javax.enterprise.inject.Produces;
+import javax.inject.Inject;
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+import java.util.List;
+import java.util.Map;
+
+/**
+ *
+ */
+public class BridgeHelper {
+
+    @ConfigProperty(name = "ob.enabled", defaultValue = "false")
+    boolean obEnabled;
+
+    @ConfigProperty(name = "ob.kcUser")
+    String kcUser;
+    @ConfigProperty(name = "ob.kcPass")
+    String kcPass;
+    @ConfigProperty(name = "ob.bridge.name")
+    String ourBridge = "heiko-bridge";
+    @ConfigProperty(name = "ob.token.user")
+    String tokenUser;
+    @ConfigProperty(name = "ob.token.pass")
+    String tokenPass;
+
+    @Inject
+    @RestClient
+    BridgeApiService apiService;
+
+    @Inject
+    @RestClient
+    BridgeAuthService authService;
+
+
+    @Produces
+    public Bridge getBridgeIfNeeded() {
+
+        if (!obEnabled) {
+            return new Bridge("- OB not enabled -", "http://does.not.exist");
+        }
+
+        if (BridgeHolder.getInstance().getBridge() != null) {
+            return BridgeHolder.getInstance().getBridge();
+        }
```

Comment:
Why is `BridgeHolder` needed? Can't we store the `Bridge` instance as a non-static field in this class after `@ApplicationScoped` has been added?